xml/Accessibility/CAccPropServices.xml

```diff
@@ -16,10 +16,10 @@
     </Interface>
   </Interfaces>
   <Attributes>
-    <Attribute>
+    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2">
```

Comment:
@joelmartinez @dend does FrameworkAlternate always show up? For these attributes for example, I wouldn't expect any FrameworkAlternate values given those monikers area basically the ones supported by this API.

Reply:
`FrameworkAlternate` shows up in any case where the attribute isn't preset on *all* monikers. In this case, the xamarin* monikers aren't present in the list, so by definition, it needs `FrameworkAlternate` :)

Reply:
Ah gotcha @joelmartinez! I thought it'd only show up when the attribute isn't present in all monikers supported for that API.